packages/@lightningjs/ui-components/src/components/InlineContent/InlineContent.stories.js

```diff
@@ -18,13 +18,11 @@
 
 import lng from '@lightningjs/core';
 import { default as InlineContentComponent } from '.';
-import mdx from './InlineContent.mdx';
 import lightningbolt from '../../assets/images/ic_lightning_white_32.png';
 import { getHexColor } from '../../utils';
-import { CATEGORIES } from '../../docs/constants';
 
 export default {
-  title: `${CATEGORIES[8]}/InlineContent`,
+  title: 'Components/InlineContent',
   args: {
     contentWrap: false,
```

Comment:
Not because of this PR, but could we set this to `true`? I feel like I always have to double check if the story is working, because I expect the line breaks to be there on first load.